safehttp/plugins/xsrf/html/xsrf.go

```diff
@@ -0,0 +1,160 @@
+// Copyright 2020 Google LLC
```

Comment:
Has this been a move too? I see that you've used `git mv` on the test. What about this one?

Reply:
I used `git add` for all the files. In the case of tests, git detected it's a move but for this one it didn't. This is probably because I moved the part that verifies a request's method is state preserving in `plugins/xsrf/xsrf.go` so, since there were slight modifications in the file, it interpreted it as a new file. Sorry about this, will use `git mv` in the future.

Reply:
No problem. Please double-check your PRs. If `git mv` hasn't existed, you could always leave a comment for the reviewers to ignore a file because it was simply moved with no changes.